core/src/testUtils/groovy/nu/yona/server/test/User.groovy

```diff
@@ -50,7 +50,7 @@ class User
 		{
 			this.nickname = json.nickname
 
-			this.buddies = (json._embedded?.buddies) ? json._embedded.buddies.collect{new Buddy(it)} : []
+			this.buddies = (json._embedded?.buddies && json._embedded.buddies._embedded) ? json._embedded.buddies._embedded.buddies.collect{new Buddy(it)} : []
```

Comment:
This can be written simpler:
```
this.buddies = (json._embedded?.buddies?._embedded) ? json._embedded.buddies._embedded.buddies.collect{new Buddy(it)} : []
```